insights/parsers/limits_conf.py

```diff
@@ -2,86 +2,16 @@
 Limits configuration - file ``/etc/security/limits.conf`` and others
 ====================================================================
 
-There are two parsers here:
-
-get_limits
-----------
-
-The ``get_limits`` function parser simply returns a dictionary with the
-domain, type, item and value of each row found in the file, indexed in the
-dictionary by the file name.
-
-LimitsConf
-----------
-
 The ``LimitsConf`` class parser, which provides a 'rules' list that is
 similar to the above but also provides a ``find_all`` method to find all the
 rules that match a given set of criteria and other properties that make it
 easier to use the contents of the parser.
 
 """
 
-import os
 from .. import Parser, parser, get_active_lines
 
 
-def parse_line(string):
-    domain, type_, item, value = string.split()
-    # Special '-1' value for unlimited, also represented as strings
-    if value == 'unlimited' or value == 'infinity':
-        value = -1
-    else:
-        value = int(value)
-    return {
-        "domain": domain,
-        "type": type_,
-        "item": item,
-        "value": value
-    }
-
-
-@parser("limits.conf")
-@parser("limits.d")
-def get_limits(context):
-    """
-    Returns a dictionary with one item, keyed to the file name, which is a
-    list of dictionaries corresponding to each rule in the file in order.
```

Comment:
@bfahr - I added the warning into the docsting, would you please check it again? Thanks.